charters/wot-wg-charter-draft-2019.html

```diff
@@ -178,39 +186,369 @@ <h2>Scope</h2>
+        </p>
+        <p>
+          Based on a list of target security mechanisms identified by
+          the WoT Interest Group, this work item will include the following tasks:
+          <ol>
+          <li>For each target security mechanism, define a standard extension
+              vocabulary for a security scheme supporting that mechanism
+              that can be used with the current TD specification.</li>
+          <li>For a core set of security schemes, propose extensions to the 
+              core TD vocabulary to include them by default.</li>
+          </ol>
+        </p>
+        <p>
+          Candidates for target security schemes include support 
+          for all flows in OAuth2, support for PoP Tokens, and support for ACE.
+        </p>
 	      
-	<h3>Item 2</h3>
-	<p>Details for item 2.</p>
+	<h3>TD Link Relation Types</h3>
+	<p>
+          The
+          <a href="http://www.w3.org/TR/wot-thing-description">WoT Thing Description (TD)</a>
+          includes a link mechanism to declare (potentially bidirectional) relationships between
+          the TD and other entities identified by URIs, including but not limited to other TDs.
+          Each link includes a relation type which specifies the purpose of the link.
+          Link relation types must use an IANA name.
+          However, to maximize interoperability, 
+          relation types for specific purposes should be identified.
+          In collaboration with the WoT Interest Group and after use-case
+          analysis, the Working Group will identify IANA relation types of
+          particular interest and specify their use for suitable use cases.
+          If a suitable IANA relation type does not exist for an identified
+          use case the Working Group may seek to establish a new link type,
+          but this will be done only if absolutely necessary.
+        </p>
+	<p>
+          Candidates for link relation type specification include part-of and subtype-of 
+          relationships to support Templates and Components, respectively.
+          Other link types may be used to identify location or controller-controlled 
+          relationships.
+        </p>
 	      
+	<h3>Architecture Use Cases and Vocabulary</h3>
+        <p>
+          The <a href="https://www.w3.org/TR/wot-architecture">WoT Architecture</a>
+          specification includes the definitions of several terms and concepts used to
+          define WoT specifications.  These are often related closely to use cases
+          and architectural patterns.  As WoT implementations mature, it will be 
+          necessary to define new use case, architectural patterns, and associated 
+          concepts and vocabulary.  In addition, descriptions of existing use cases
+          architectural patterns may need to be clarified or refactored to include
+          these new concepts.
+        </p>
+	<p>
+          Candidates for architectural vocabulary extension include a Producer concept 
```

Comment:
Producer of TDs?

Reply:
I changed this to "TD Producer" to be clearer.